docs/cql/ddl.rst

```diff
@@ -897,6 +912,46 @@ The following modes are available:
    * - ``immediate``
      - Tombstone GC is immediately performed. There is no wait time or repair requirement. This mode is useful for a table that uses the TWCS compaction strategy with no user deletes. After data is expired after TTL, ScyllaDB can perform compaction to drop the expired data immediately.
 
+.. _cql-hints-options:
+
+Table hints
+###########
+
+The per-table ``hints`` options can be used to tune the tablet allocation logic for the table.
+
+=============================== =============== ===================================================================================
+ Option                          Default         Description
+=============================== =============== ===================================================================================
+ ``expected_data_size_in_gb``   0               The expected table data size, in combination with the ``target_tablet_size_in_bytes``
+                                                config option, the replication factor configuration, and the cluster topology
+                                                determine the minimum number of tablets to allocate for the table.
+                                                It can be set when the table is created to allocate more tablets for it,
+                                                as if it already occupies that size.
+                                                It can also be used later in the tablet life cycle to induce tablet splits or merges
+                                                to fit the new expected size.
+ ``hot_table``                  'false'         Hint that this table expects relatively higher traffic than other tables so that the
+                                                tablet allocation algorithm should give it higher priority for balancing its tablets.
```

Comment:
shouldn't this be shares instead? a binary prevents the possibility of declaring different degrees of hotness. it can become a escape hatch (setting statically) once we have shares calculated dynamically by the balancer.

Reply:
We discussed having shares as a measure, but it seems more complicated for the user.

Need be, min_per_shard_tablet_count can be used, and it is very similar to shares, just that the maximum number (100) is clearer and would be configurable, even per node, to accommodate heterogeneous nodes with different-capacity shards

Reply:
Will be dropped in the next version and be reconsidered in a followup change.